webapp-django/crashstats/crashstats/utils.py

```diff
@@ -560,7 +560,7 @@ def build_default_context(product_name=None, versions=None):
     except productlib.ProductDoesNotExist:
         raise http.Http404("Not a recognized product")
 
-    context["product"] = product.name
+    context["product"] = product
```

Comment:
Now "product" is a Product instance. That simplifies some templates and other things.